VespaGAS/ViewModels/GasCalculatorViewModel.swift

```diff
@@ -1,5 +1,36 @@
 import Foundation
+import RxSwift
 
-public struct GasCalculatorViewModel {
+struct GasCalculatorViewModel {
+    private let disposeBag = DisposeBag()
+    private let oilValueSubject = BehaviorSubject(value: "0%")
+    private let gasValueSubject = BehaviorSubject(value: "0L")
 
+    var oilMixObservable: Observable<String> = Observable.empty()
+    var oilValueObservable: Observable<String> {
+        return oilValueSubject.asObservable()
+    }
+    var gasValueObservable: Observable<String> {
+        return gasValueSubject.asObservable()
+    }
+
+    init(gasObservable: Observable<GasType>, oilPercentageObservable: Observable<OilMixType>) {
+        oilMixObservable = Observable.combineLatest(gasObservable, oilPercentageObservable) { (gasInLiter, oilPercentage) -> String in
+            if let oilValue = OilMixCalculator.oilMLForGasoline(gasInLiter, oilPercentage: oilPercentage) {
```

Comment:
this is some logic that is unrelated to the view and to be doesn't make sense to be in a view model. This is actually the _model_ in my opinion. Might be worth splitting it out?